src/routes/lobby/LobbyView.vue

```diff
@@ -121,7 +119,14 @@ export default {
   setup() {
     const { t } = useI18n();
     const gameStarted = ref(false);
-
+    const joinAudio = new Audio('/sounds/lobby/enter-lobby.mp3');
+    const leaveAudio =new Audio('/sounds/lobby/leave-lobby.mp3');
+    onMounted(() => {
+      joinAudio.play();
+    });
+    onUnmounted(() => {
+      leaveAudio.play();
```

Comment:
Previously we had checks that the audio file was ready to play before playing it. What happens if this is called before the audio file is downloaded and ready to play?